scripts/update_repos_version.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+import os
+import os.path
+import re
+import logging
+import subprocess
+import fileinput
+import shutil
+import click
+import github
+
+LOG_FILENAME = 'bump_repo_version_log.txt'
+logging.basicConfig(filename=LOG_FILENAME,level=logging.INFO)
```

Comment:
I generally prefer logging to stdout and saving it to file if necessary via `tee` or '>', or default piping to both.